host/sgx/sgxload.c

```diff
@@ -538,6 +573,8 @@ oe_result_t oe_sgx_load_enclave_data(
     bool extend)
 {
     oe_result_t result = OE_UNEXPECTED;
+    uint64_t base_address =
```

Comment:
Added enclave->start_addr and enclave->base_addr fields and passing in oe_enclave_t instead of enclave_addr in the functions that add data and control pages.